mlflow/server/js/src/model-registry/services.js

```diff
@@ -111,7 +111,7 @@ export class Services {
    * @returns {*|jQuery|*|*}
    */
   static setRegisteredModelTag({ data, success, error }) {
-    return $.ajax(Utils.getAjaxUrl('ajax-api/2.0/preview/mlflow/registered-models/set-tag'), {
```

Comment:
In contrast with https://github.com/mlflow/mlflow/pull/3438/files#r491200592, we opt to immediately update UI logic to hit non-preview endpoints because users are very unlikely to run an updated version of the MLflow UI against an older server (both of these are generally exposed on / managed by the same host).